.github/workflows/linter.yml

```diff
@@ -47,3 +47,4 @@ jobs:
           VALIDATE_ALL_CODEBASE: true
           VALIDATE_JAVASCRIPT_STANDARD: false
           VALIDATE_JSCPD: false
+          VALIDATE_GITHUB_ACTIONS: false
```

Comment:
Since this is temporary until the issue you mentioned is completed, thoughts on putting a comment with a `TODO`?

`TODO: Disable GitHub action lint until https://github.com/rhysd/actionlint/pull/418 is released`